source/base/function_lib.cc

```diff
@@ -2606,7 +2606,7 @@ namespace Functions
     const Point<dim>  &p,
     const unsigned int component) const
   {
-    (void)component;
```

Comment:
and simply remove the `(void)component` line.

Reply:
Oh ...Thanks!